src/presentation/pages/Note.vue

```diff
@@ -46,6 +52,23 @@ function noteChanged(data: NoteContent): void {
     save(data);
   }
 }
+
+/**
+ * Changing the title in the browser
+ */
+if (!props.id) {
+  useHead({
+    title: t('site.titles.newNote'),
+  });
+} else {
+  watchEffect(() => {
```

Comment:
useHead is not reactive?

Reply:
At the moment we decided to leave this function watchEffect